pkg/vm/stackitem/serialization.go

```diff
@@ -23,14 +23,30 @@ func SerializeItem(item Item) ([]byte, error) {
 // similar to io.Serializable's EncodeBinary, but works with Item
 // interface.
 func EncodeBinaryStackItem(item Item, w *io.BinWriter) {
-	serializeItemTo(item, w, make(map[Item]bool))
+	serializeItemTo(item, w, false, make(map[Item]bool))
 }
 
-func serializeItemTo(item Item, w *io.BinWriter, seen map[Item]bool) {
+// EncodeBinaryStackItemAppExec encodes given Item into the given BinWriter. It's
+// similar to EncodeBinaryStackItem but allows to encode interop (only type, value is lost).
+func EncodeBinaryStackItemAppExec(item Item, w *io.BinWriter) {
+	bw := io.NewBufBinWriter()
+	serializeItemTo(item, bw.BinWriter, true, make(map[Item]bool))
+	if bw.Err != nil {
+		w.WriteBytes([]byte{byte(InvalidT)})
```

Comment:
We may probably want to set bw.Err to nil after that.

Reply:
No need for this: it is local to this function and there is immediate return after.